packages/node/src/utils/meta.ts

```diff
@@ -49,17 +57,17 @@ export function getTracingMetaTags(
 
   const isValidSentryTraceHeader = TRACEPARENT_REGEXP.test(sentryTrace);
   if (!isValidSentryTraceHeader) {
-    logger.warn('Invalid sentry-trace data. Returning empty <meta name="sentry-trace"/> tag');
+    logger.warn('Invalid sentry-trace data. Returning empty "sentry-trace" meta tag');
```

Comment:
l: Just to be more clear that we aren't actually emitting serialized meta tags. Feel free to disregard tho.

```suggestion
    logger.warn('Invalid sentry-trace data. Returning empty "sentry-trace" meta tag data');
```